### PR TITLE
Added new step templates to register and unregister ETW manifests using wevtutil

### PR DESCRIPTION
**Pull request by HumanPrinter**

### CODE REVIEW

File: step-templates/event-tracing-register-manifest.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "Properties": {
+    "Octopus.Action.Script.Syntax": "PowerShell",
+    "Octopus.Action.Script.ScriptBody": "if(-not (Test-Path $env:SystemRoot\\System32\\wevtutil.exe))\n{\n    throw \"wevtutil.exe could not be found\"\n}\n\nif(-not (Test-Path $ManifestFile))\n{\n    throw \"Manifest $manifest could not be found\"\n}\n\nif(-not (Test-Path $ResourceFile))\n{\n    throw \"Resource file $ResourceFile could not be found\"\n}\n\nif(-not (Test-Path $MessageFile))\n{\n    throw \"Message file $MessageFile could not be found\"\n}\n\n& \"$env:SystemRoot\\System32\\wevtutil.exe\" im $ManifestFile /rf:$ResourceFile /mf:$MessageFile"
+  },
+  "SensitiveProperties": {},
+  "Parameters": [
+    {
+      "Name": "ManifestFile",
+      "Label": "Manifest file",
+      "HelpText": "Full path to the manifest file that must be unregistered.",
```
Comment (alfhenrik) [inline step-templates/event-tracing-register-manifest.json]: Should `unregistered` here be `registered`?

**Comment by HumanPrinter** [comment]
You are absolutely right. I made a second commit to correct the problem

File: step-templates/event-tracing-register-manifest.json (hunk 1 above)
Comment (HumanPrinter) [inline step-templates/event-tracing-register-manifest.json]: You are absolutely right. I made a second commit to correct the problem

**Pull request merged**